lib/rhc/ssh_helpers.rb

```diff
@@ -461,9 +461,11 @@ def discover_ssh_executable
           # looks for ssh.exe from msysgit or plink.exe from PuTTY, either on path or specific locations
           guessing_locations <<
             discover_windows_executables do |base|
-              [
+              from_path = ENV['PATH'].split(File::PATH_SEPARATOR).map {|p| p + File::ALT_SEPARATOR + "ssh.exe" }
```

Comment:
Wouldn't it make more sense to add `ENV['PATH'].split(File::PATH_SEPARATOR)` to `base_files ` in `discover_windows_executables`?